specification/schemas/Action.json

```diff
@@ -0,0 +1,48 @@
+{
+  "type": "object",
+  "required": [
+    "action_type",
+    "values"
+  ],
+  "additionalProperties": false,
+  "properties": {
+    "action_type": {
+      "type": "string",
+      "enum": [
+        "create-resource",
+        "update-resource"
+      ],
+      "example": "create-resource"
+    },
+    "resource_type": {
+      "type": "string",
+      "example": "services"
+    },
+    "values": {
+      "type": "array",
+      "items": {
+        "oneOf": [
+          {
+            "$ref": "#/components/schemas/PredicateValue"
+          },
+          {
+            "$ref": "#/components/schemas/ScalarValue"
+          },
+          {
+            "$ref": "#/components/schemas/PathValue"
+          },
+          {
+            "$ref": "#/components/schemas/ResourceValue"
+          }
```

Comment:
If you look at the proposal, the values here aren't just values, but have a `path` and then a `value` to indicate what value should be assigned to what value.